### ISSUE TITLE
[Bug]: Incorrect json in ide.json result can't opening project in phpstorm

### ISSUE THREAD

**Issue by alexandre-tobia** [laravel-idea/plugin#541]
### Bug description

Hi,

When i use an ide.json with incorrect json the project won't load at all.

Example:
```json
{
  "$schema": "http://laravel-ide.com/schema/laravel-ide-v1.json",
  "view": {
    "paths": [
      {
        "path": "web/app/themes/casden/resources/views",
        "namespace": ""
      }
    ]
  },
  "blade": {
    "components": {
      "viewPaths": [
        {
          "path": "web/app/themes/casden/resources/views/components",
         "namespace": ""
        }, <==== HERE
      ]
    }
  }
}
```
Please note the extra comma after the viewPaths properties (juste before the closing **]** )
If i don't remove it, i can't open the project in PHPstorm (is automatically closed by the IDE without any error)

Maybe add an extra check to verify if the json is valid before trying to load it ? 

### Plugin version

5.3.0.221

### Operating system

MacOS

### Steps to reproduce

Create an ide.json with an invalid json and try to close phpstorm and load the project again.

**Comment by adelf** [comment]
Thanks for posting. This is weird. Couldn't reproduce and the code parsing this JSON looks like that:
```
try {
    gson.fromJson(it.inputStream.reader(), ConfigModel::class.java)
} catch (e: Throwable) {
    ConfigModel.EMPTY
}
```

So it shouldn't give any exceptions... Do you have anything in the PhpStorm log? main menu > Help > Show log in Finder.

**Comment by alexandre-tobia** [comment]
[idea.log](https://github.com/laravel-idea/plugin/files/8529886/idea.log)

Here we go. I don't know if this is helpful or not.

**Comment by adelf** [comment]
Got it now. Thanks. Anyway it shouldn't close PhpStorm... But I'll try to fix it asap.

**Comment by alexandre-tobia** [comment]
It don't close phpstorm but i can't open the projet afterwards. It does nothing when i click on the project name on the launcher. I need to invalidate the cache + fix the json before trying to open the project again.
Here:
![Capture d’écran 2022-04-21 à 13 09 39](https://user-images.githubusercontent.com/5537783/164445767-fafcbc30-6f19-4702-97c0-69805c394922.jpg)

Thanks for your quick response !

**Comment by adelf** [comment]
Should be fixed by the 5.3 version.

**Comment by alexandre-tobia** [comment]
@adelf Thanks !
